doc_files/iter8_bookinfo_istio.md

```diff
@@ -406,7 +406,7 @@ In this tutorial, we will define a custom metric (one not provided by _iter8_ ou
 By default _iter8_ provides a few metrics which you can see if you type:
 
 ```bash
-$ kubectl get configmap iter8config-metrics -n iter8 -oyaml
```

Comment:
All other bash commands do not have the preceding `$`.

Only shell session examples use the `$`.